packages/reports/addon/components/dropdown-parameter-picker.hbs

```diff
@@ -0,0 +1,13 @@
+{{!-- Copyright 2021, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<PowerSelect
+  @options={{this.options}}
+  @onOpen={{this.fetchParameterOptions}}
```

Comment:
Instead of kicking off an action, can we pass an action that returns a promise of options to the `@options` arg here

Reply:
So I've tried this and got Kevins recommendation, when using an action in @options the dropdown shows no results.

Reply:
@holderb5 On second thought, we probably don't want to fetch until there is an open event. No need to fetch the values if you are not going to use them :) But for the record, this is how you do pass a promise to power-select:
```ts
  get parameterOptions() {
    return (async () => {
      const values = await this.args.columnMetadata.parameters[this.args.parameterIndex].values;
      return [
        {
          groupName: this.parameterName,
          options: values?.map((el) => el.id),
        },
      ];
    })();
  }
```